### DISCUSSION TITLE
识别结果为乱码时运行时间会超级慢

### DISCUSSION THREAD

**Discussion by with-him777** [OleehyO/TexTeller#84] [Q&A]
![1](https://github.com/user-attachments/assets/24706275-5fa0-4db3-a485-bfafd18d9759)
<img width="1083" alt="截屏2024-08-06 15 35 47" src="https://github.com/user-attachments/assets/4e1deaca-ab3b-48cd-ac41-1042a4c98537">
这种图片会运行80多秒，而且会导致显存爆炸而报错，请问一下有什么好的办法提前终止或者速度提升吗

**Comment by OleehyO** [comment]
huggingface的.generate方法应该有提供这样的配置，可以参考一下。
